vllm/worker/hpu_model_runner.py

```diff
@@ -1241,7 +1246,6 @@ def move_model_to_hpu(model):
                         os.getenv("QUANT_CONFIG", ""))
                     self._inc_preprocess()
```

Comment:
can you double check if "self._inc_preprocess()" need run on cpu model or hpu is ok?